less/core/notify.less

```diff
@@ -29,6 +29,28 @@
     .icon-warning;
   }
 
+  &__popup.has-graphic {
+    .notify__section-inner {
+      display: flex;
+    }
+
+    &.is-graphic-top .notify__section-inner,
+    &.is-graphic-bottom .notify__section-inner {
+      flex-direction: column;
+      row-gap: 1rem;
+    }
+
+    &.is-graphic-left .notify__section-inner,
+    &.is-graphic-right .notify__section-inner {
+      flex-direction: row;
+      column-gap: 1rem;
+      .notify__body,
+      .notify__image-container {
+        width: 50%;
```

Comment:
For consistency with other popup content ([Hotgraphic](https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/a34c6b7973d8015a588faae080c9baaf0b438ec6/less/hotgraphicPopup.less#L48), Hotgrid etc) should we make this a 60/40 layout when graphic is left/right aligned?

Also like Hotgraphic popup, should the [text and image stack for mobile view](https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/a34c6b7973d8015a588faae080c9baaf0b438ec6/less/hotgraphicPopup.less#L42)? Without stacking, this fits approx. 3 words per line and this will be less for lengthier languages.